projects/core/src/lib/backend/backend.service.ts

```diff
@@ -38,15 +38,36 @@ import {
     LayerDict,
     LayerCollectionDict,
     AuthCodeRequestURL,
+    BackendInfoDict,
 } from './backend.model';
 
+export interface BackendStatus {
+    available: boolean;
+    httpError?: HttpErrorResponse;
+    initial?: boolean;
+}
+
 @Injectable({
     providedIn: 'root',
 })
 export class BackendService {
     readonly wmsBaseUrl = `${this.config.API_URL}/wms`;
+    protected readonly backendStatus$ = new BehaviorSubject<BackendStatus>({available: false, initial: true});
```

Comment:
Please, don't put any state into this service.
This should plainly wrap the backend API and provide method to call funcitons.

Morover, I hope someday to replace this with auto generated OpenAPI stuff.

Reply:
[ff4e8dc](https://github.com/geo-engine/geoengine-ui/pull/512/commits/ff4e8dcad6caa7013502f09cad6bfe23ae30690b)